.github/workflows/appengine-deploy-it.yml

```diff
@@ -10,52 +15,66 @@ jobs:
       with:
         service_account_key: ${{ secrets.APPENGINE_DEPLOY_SA_KEY_JSON }}
         export_default_credentials: true
+    - name: create yaml
+      run: |-
+        cat <<EOF > ${{ github.workspace }}/app.yaml
+        service: "${{ github.job }}-${{ github.run_number }}"
```

Comment:
This should be "unique enough"

Reply:
seems like GAE cannot have underscores in the service name and one of the `github.job` is `b64_json`.
https://github.com/GoogleCloudPlatform/github-actions/pull/123/checks?check_run_id=747208506

Reply:
Done